### PR TITLE
updating office hours

### PR DESCRIPTION
**Pull request by chrislovecnm**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1632)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,11 +93,14 @@ to working with newcomers, helping with PRs, and discussing new features.
...
-|--------------|---------------|-------|
-| [@justinsb](https://github.com/justinsb)             |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
-| [@chrislovecnm](https://github.com/chrislovecnm)     |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
-| [@kris-nova](https://github.com/kris-nova)           |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
+Office hours are every other week, on odd week numbers. Easy command for number
+of the week: `date +%V`.
+
+| Maintainer   |  URL |
+|--------------|-------|
+| [@justinsb](https://github.com/justinsb)             | [Zoom](https://zoom.us/my/k8ssigaws) |
```
Comment (justinsb) [inline README.md]: We probably need the time in here

Comment (kris-nova) [inline README.md]: We probably need a kops mailing list so we can do the same thing we do with sig-aws :smile:

Comment (kris-nova) [inline README.md]: ..err.. just use a sig's mailing list to track it..

Comment (kris-nova) [inline README.md]: But yes - please at the time :)

Comment (chrislovecnm) [inline README.md]: Is there a reason our names are there?? This is community :) I am going to add the time, the zoom URL and pull our names :)

**Comment by chrislovecnm** [comment]
can I get a /lgtm?

File: README.md [hunk 2]
```diff
@@ -90,17 +90,14 @@ Kops also has time set aside every other week to offer help and guidance to the
 community. Kops maintainers have agreed to set aside time specifically dedicated
 to working with newcomers, helping with PRs, and discussing new features.
 
-We recommend letting us know **beforehand** if you plan on attending so we can
-have time to prepare for the call.
+Please feel free to reach out **beforehand** if you plan on attending!
 
-| Maintainer   | Schedule      |  URL |
```
Comment (kris-nova) [inline README.md]: Why are we nuking the maintainers here? Just curious. I think it is important to let people know who they can expect to see on the call, also if they need to ping someone before/after

Comment (chrislovecnm) [inline README.md]: I made a comment that we are not we. We are contributors, but we are community. I don't want people to look at us and compare themselves to us. Inclusion ... I know we are awesome. I don't think we need to broadcast it. K8s core does not have it, we are not 3, we are many.

Prime time

Go team!!!!

Comment (kris-nova) [inline README.md]: Fine with removing it, but in general the maintainers are the ones expected to be on the call. Maintainers have different expectations from the community, and if you made the list you should be held to your commitment as a maintainer (in writing) IMHO.. so LGTM on this for now.. but would like to see a maintainers list pop up somewhere else if we could..

Comment (kris-nova) [inline README.md]: Basically.. if we call the maintainers out that they have different expectations than the community for the call.. then we need to call out who the exclusions are for..

File: README.md [hunk 3]
```diff
@@ -90,17 +90,14 @@ Kops also has time set aside every other week to offer help and guidance to the
...
-|--------------|---------------|-------|
-| [@justinsb](https://github.com/justinsb)             |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
-| [@chrislovecnm](https://github.com/chrislovecnm)     |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
-| [@kris-nova](https://github.com/kris-nova)           |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
+Office hours, on [Zoom](https://zoom.us/my/k8ssigaws) video
+conference are on Fridays at 9 am PT every other week, on odd week numbers.
```
Comment (tazjin) [inline README.md]: This should mention the time in UTC (i.e. 17:00 / 5PM UTC) and contain a link to [this view](http://www.worldtimebuddy.com/?pl=1&lid=100,5,8,2950159&h=100) which highlights the most important EU/US timezones & UTC and gives users the ability to easily add their own local timezone to the view.

Comment (chrislovecnm) [inline README.md]: @tazjin you have a good site we can link?

Comment (tazjin) [inline README.md]: Yes, see the link in my previous comment :)

**Comment by chrislovecnm** [comment]
We need to have a different URL for the Zoom meeting.  http://everytimezone.com/#2017-2-3,300,cn3

**Pull request merged**